packages/ui/src/memberships/components/MemberList/Members.tsx

```diff
@@ -28,7 +31,9 @@ export const MemberList = ({ members, order, onSort }: MemberListProps) => {
         )
       : ListHeader
 
-  return (
+  return isLoading ? (
+    <Loading />
+  ) : (
```

Comment:
Could you unpack `?`. I think it will provide more readability:

```typescript
if (isLoading) {
	return <Loading />
}

return (
  <div>...
```